src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -856,7 +856,7 @@ def DeviceAvailableCallback(device, err):
                     raise TimeoutError("Timed out waiting for DNS-SD resolution")
 
         if returnDevice.value is None:
-            raise self._ChipStack.ErrorToException(returnErr)
+            returnErr.success_or_raise()
```

Comment:
A quick search reveals a lot more `ErrorToException` calls that remain. Shouldn't those get converted?

Reply:
https://github.com/project-chip/connectedhomeip/pull/22224#issuecomment-1282527681

As commented below, mark as resolved.